net.sf.eclipsecs.checkstyle/.classpath

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry exported="true" kind="lib" path="checkstyle-8.35-all.jar" sourcepath="checkstyle-checkstyle-8.35.zip"/>
+	<classpathentry exported="true" kind="lib" path="checkstyle-8.36-SNAPSHOT-tests.jar" sourcepath="checkstyle-checkstyle-8.35.zip"/>
```

Comment:
You would need to provide the correct zip file for the sourcepath, or you can try not providing a sourcepath.